### DISCUSSION TITLE
where they go?

### DISCUSSION THREAD

**Discussion by sharkmullet** [Extended-Thunder/send-later#387] [Q&A] [answered]
Thunderbird just updated, and send later seems to be working...but the messages always went into my drafts folder.
Sometimes I compulsively like to check the messages and see them in the drafts waiting to go out.  Now I can't find them, I see the bar on the bottom saying "Send Later [PEND 1]", but I don't see it in my drafts box.  I also checked the 'outbox' and don't see it there.  Where might it be???

**Comment by jph-sendlater** [comment, marked as answer]
They're in a Drafts folder somewhere. It could be that you have Thunderbird configured to store drafts in a non-standard location. Check your account settings. Also, try composing a message and using `Ctrl+S` to save it directly to Drafts. That might help you track down where the message is being saved.

**Reply by sharkmullet** [reply]
I tried ctrl+s and it went to the drafts folder where I expected.  Send laters don't go there though still

**Reply by sharkmullet** [reply]
well, now it is! Not sure what changed

**Reply by sharkmullet** [reply]
I think I had accidentally sorted the drafts folder by some method other than the date, so it wasn't at the bottom. Yikes.  Thank you
